marklogic-client-api/src/test/java/com/marklogic/client/test/ssl/TwoWaySSLTest.java

```diff
@@ -96,15 +90,19 @@ public static void teardown() {
 	 * SSLContext can connect to the app server.
 	 */
 	@Test
-	void digestAuthentication() throws Exception {
+	void digestAuthentication() {
 		if (Common.USE_REVERSE_PROXY_SERVER) {
 			return;
 		}
 
 		// This client uses our Java KeyStore file with a client certificate in it, so it should work.
 		DatabaseClient clientWithCert = Common.newClientBuilder()
+			.withKeyStorePath(keyStoreFile.getAbsolutePath())
```

Comment:
This is the main value of this story - the user only needs to provide a couple properties for 2-way SSL now instead of all the annoying code for building their own SSL context with all the proper error handling.